routes/api/contacts.js

```diff
@@ -1,27 +1,20 @@
-const express = require('express')
-const contacts = require("../../models/contacts")
+const express = require('express');
 
+const { validation } = require('../../middlewares');
+const { contactSchema } = require('../../shemas');
+const { contacts: ctrl } = require('../../controllers/');
```

Comment:
```suggestion
const { contacts: { listContacts, } } = require('../../controllers/');
```
деструктуризацію можна продовжувати